### DISCUSSION TITLE
绳网研学

### DISCUSSION THREAD

**Discussion by zgyhim** [share121/inter-knot#5758] [公告]
![Screenshot_20241214_142536_edit_39599652862706](https://github.com/user-attachments/assets/4340e24d-1da0-4e7e-9aa5-e767363e84b6)
![Screenshot_20241214_142603_edit_39761848115806](https://github.com/user-attachments/assets/c2414992-64b2-4e9c-81d0-f57814852d6b)
![Screenshot_20241214_142526_edit_39507905593970](https://github.com/user-attachments/assets/0c6ac414-52ba-4e11-a8fe-c9e24eec3eb1)
![Screenshot_20241214_142549_edit_39706232274669](https://github.com/user-attachments/assets/d3594594-0ea2-4461-900c-ec02f549eb1f)

**纳尼 绳网搞起研学了?**

**Comment by luckylca** [comment]
nb

**Comment by MO-hy** [comment]
豆！

**Comment by Myitian** [comment]
![图片](https://github.com/user-attachments/assets/1b4735d4-de77-4a07-8ef0-67c8e404c826)
![doge](https://i0.hdslb.com/bfs/emote/3087d273a78ccaff4bb1e9972e2ba2a7583c9f11.png)
